recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   noarch: python
-  number: 1
+  number: 2
```

Comment:
What's the significance of this?

Reply:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build-number-and-string

since nothing is changed in the package (same version) but there are changes in the recipe, the build number has to be increased. Don't forget to set it to 0 again with the next version bump.